assets/src/scripts/components/SearchBar/SearchBarResult.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+/**
+ * History List.
+ */
+export default class SearchBar_Result extends React.Component {
+
+	render() {
+
+		var classes = [ 'SearchBar_Result' ];
```

Comment:
that's correct, I just interpreted these as prefer `let` over `var` https://gist.github.com/rmccue/8da274a14a03207849e4c1356255ed6a#variable-declaration , maybe I'm wrong